docs/subvolume.md

```diff
@@ -0,0 +1,55 @@
+# Subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which have no parent-pvc attached to them.
```

Comment:
Did we consider the external ceph cluster case to identify the stale subvolumes?

Reply:
No, we need the cluster setup to check it. I did ping ashish , waiting for him on that part.
https://github.com/rook/kubectl-rook-ceph/issues/211 this is the issue created to complete te remaining scenarios and minor changes .